src/platform/packages/shared/kbn-unified-data-table/src/components/data_table_document_selection.tsx

```diff
@@ -33,6 +33,7 @@ import { UnifiedDataTableContext } from '../table_context';
 import { DataTableCopyRowsAsText } from './data_table_copy_rows_as_text';
 import { DataTableCopyRowsAsJson } from './data_table_copy_rows_as_json';
 import { useControlColumn } from '../hooks/use_control_column';
+import { type CustomBulkActions } from './data_table';
```

Comment:
And import from `types.ts` file here.

Reply:
fixed in https://github.com/elastic/kibana/pull/230738/commits/bd8d541efe395f5cc2ce52d6619b345635b74f29